lib/web/apiserver.go

```diff
@@ -771,8 +771,13 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.GET("/webapi/auth/export", h.authExportPublic)
 
 	// join token handlers
-	h.PUT("/webapi/token/yaml", h.WithAuth(h.upsertTokenContent))
-	h.POST("/webapi/token", h.WithAuth(h.createTokenHandle))
+	h.PUT("/webapi/token/yaml", h.WithAuth(h.editTokenYAML))
+	// used for creating a new token
+	h.POST("/webapi/token/new", h.WithAuth(h.upsertTokenHandle))
```

Comment:
Should we have `/new` in the URL? I don't see any other endpoint that uses this convention

Reply:
I can think of something better than /new, sure

Reply:
i just made it plural "/webapi/tokens", as posting to a REST api should end in the plural of the resource anyway.